src/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -389,14 +390,18 @@ public void TestUserCredentialsPropertiesOnWindows()
             }
             finally
             {
+                Interop.NetUserDel(null, username);
```

Comment:
NetUserDel returns a status code.  Can we check that to validate that the user was deleted successfully?